src/models/gateway-account/GatewayAccountSwitchPaymentProviderRequest.class.ts

```diff
@@ -0,0 +1,28 @@
+interface GatewayAccountSwitchPaymentProviderRequestData {
+  user_external_id: string | undefined
```

Comment:
Why `user_external_id: string | undefined` as opposed to `user_external_id?:  string`?

Reply:
they accomplish the same thing, just a style choice. Will update